tests/test_file_generation.py

```diff
@@ -193,77 +181,64 @@ def test_different_roots_graph() -> None:
-            ),
-            [
-                "/SPDX Lite Document/DESCRIBES/Package A/CONTAINS/",
-                "/SPDX Lite Document/DESCRIBES/Package A/COPY_OF/"
-                "Package C/CONTAINS/",
-            ],
-        ),
-    ],
-)
-def test_tree_generation_for_bigger_examples(
-    file_name: str,
-    expected_top_level_keys: int,
-    expected_file_path_level_1: Tuple[str, str, str],
-    expected_file_path_level_2: Tuple[str, str, str, str, str],
-    expected_breakpoints: List[str],
-) -> None:
-    document = parse_file(str(Path(__file__).resolve().parent / "data" / file_name))
-    graph = generate_graph_from_spdx(document)
-    tree = generate_tree_from_graph(graph)
-    opossum_information = generate_json_file_from_tree(tree)
-
+def test_tree_generation_for_bigger_examples_json() -> None:
+    opossum_information = file_name_to_opossum_information(
+        "SPDXJSONExample-v2.3.spdx.json"
+    )
     file_tree = opossum_information.resources.to_dict()
+
+    expected_breakpoints = [
+        "/SPDX-Tools-v2.0/CONTAINS/glibc/CONTAINS/"
+        "lib-source/commons-lang3-3.1-sources.jar/GENERATED_FROM/",
+        "/SPDX-Tools-v2.0/CONTAINS/glibc/DYNAMIC_LINK/",
+    ]
+
     assert isinstance(file_tree, dict)
-    assert len(file_tree.keys()) == expected_top_level_keys
+    assert len(file_tree.keys()) == 3
+
+    for attribution_breakpoint in expected_breakpoints:
+        assert attribution_breakpoint in opossum_information.attributionBreakpoints
     assert (
-        file_tree[expected_file_path_level_1[0]][expected_file_path_level_1[1]][
-            expected_file_path_level_1[2]
+        file_tree["SPDX-Tools-v2.0"]["COPY_OF"][
+            "DocumentRef-spdx-tool-1.2:SPDXRef-ToolsElement"
         ]
         == 1
     )
+
     assert (
-        file_tree[expected_file_path_level_2[0]][expected_file_path_level_2[1]][
-            expected_file_path_level_2[2]
-        ][expected_file_path_level_2[3]][expected_file_path_level_2[4]]
-        == 1
+        file_tree["SPDX-Tools-v2.0"]["CONTAINS"]["glibc"]["DYNAMIC_LINK"]["Saxon"] == {}
     )
 
+
+def test_tree_generation_for_bigger_examples_spdx() -> None:
+    opossum_information = file_name_to_opossum_information("SPDX.spdx")
+    file_tree = opossum_information.resources.to_dict()
+    expected_breakpoints = [
+        "/SPDX Lite Document/DESCRIBES/Package A/CONTAINS/",
+        "/SPDX Lite Document/DESCRIBES/Package A/COPY_OF/" "Package C/CONTAINS/",
+    ]
+
+    assert isinstance(file_tree, dict)
+    assert len(file_tree.keys()) == 2
+
     for attribution_breakpoint in expected_breakpoints:
         assert attribution_breakpoint in opossum_information.attributionBreakpoints
+
+    assert file_tree["SPDX Lite Document"]["DESCRIBES"]["Package B"] == {}
+
+    assert (
+        file_tree["SPDX Lite Document"]["DESCRIBES"]["Package A"]["CONTAINS"]["File-C"]
+        == 1
+    )
+
+
+def file_name_to_opossum_information(file_name: str) -> OpossumInformation:
```

Comment:
rename to `_get_opossum_information_from_file` maybe to make it a verb.
(same for the function below)